frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -129,10 +145,12 @@ export const CurrentVersion: React.SFC<CurrentVersionProps> = ({cv}) => {
   if (status === ClusterUpdateStatus.UpToDate || status === ClusterUpdateStatus.UpdatesAvailable) {
     return desiredVersion
       ? <span className="co-select-to-copy">{desiredVersion}</span>
-      : <React.Fragment><YellowExclamationTriangleIcon />&nbsp;Unknown</React.Fragment>;
+      : <><YellowExclamationTriangleIcon />&nbsp;Unknown</>;
```

Comment:
Nit:  you're using `<React.Fragment>` above (as are the previously existing examples).